src/qt/radiobut.cpp

```diff
@@ -39,11 +42,50 @@ bool wxRadioButton::Create( wxWindow *parent,
              const wxString& name)
 {
     m_qtRadioButton = new QRadioButton( parent->GetHandle() );
+
+    if ( (style & wxRB_GROUP) || (style & wxRB_SINGLE) )
+    {
+        JoinNewGroup();
```

Comment:
After the latest changes, this function which only has 2 lines can be avoided. The drawback would be 1 extra line of code here. The positive side would be the removal of the declarations of 1 function and 1 member variable plus the definition of 1 function.